### PR TITLE
PP-4918 - update copy with new go live steps

### PR DESCRIPTION
**Pull request by jonheslop**
Preview — https://build-d23yqetoh.now.sh/getstarted/

### CODE REVIEW

**Comment by jonheslop** [comment]
Preview — https://build-d23yqetoh.now.sh/getstarted/

File: source/getstarted.html.erb [hunk 1]
```diff
@@ -61,30 +61,47 @@ title: Get started
           </div>
 
           <h2 class="task-list-section">
-              <span class="task-list-section-number">5.</span> Sign GOV.UK Pay’s agreements
+              <span class="task-list-section-number">5.</span> Agree to GOV.UK Pay’s agreements
           </h2>
           <div class="task-list-items">
-            <p>Your organisation needs to sign GOV.UK Pay’s Memorandum of Understanding and your team needs to agree to our Terms of Use.</p>
-            <p>Contact GOV.UK Pay on <a href="mailto:govuk-pay-support@digital.cabinet-office.gov.uk" data-click-events data-click-category="Content" data-click-action="Email link clicked">govuk-pay-support@digital.cabinet-office.gov.uk</a> to receive our agreements.</p>
+              <p>Your organisation needs to agree to GOV.UK Pay’s Memorandum of Understanding (for crown bodies) or Contract (for non-crown bodies). They include our Terms of Use.</p>
```
Comment (alexbishop1) [inline source/getstarted.html.erb]: Capital “C” for “Crown” please

File: source/getstarted.html.erb [hunk 2]
```diff
@@ -61,30 +61,47 @@ title: Get started
...
-              </div>
-              <p>When you have a Payment Service Provider, you need to make sure it is set up to work with GOV.UK Pay. We have instructions for <a href="https://docs.payments.service.gov.uk/#switching-to-production" data-click-events data-click-category="Content" data-click-action="Internal link clicked">configuring your account</a> in our documentation. </p>
-              <p><a href="/support" data-click-events data-click-category="Content" data-click-action="Internal link clicked">Contact GOV.UK Pay</a> if you’d like more advice on contracting.</p>
+              <p>You’ll need to have a contract with a Payment Service Provider (PSP), who will process the payments taken using GOV.UK Pay. There are 3 ways to get a PSP.</p>
+              <h3 class="heading-small">1. GOV.UK Pay has procured, Stripe, as our PSP.</h3>
```
Comment (alexbishop1) [inline source/getstarted.html.erb]: Can’t say “GOV.UK Pay has procured, Stripe, as our PSP” — it doesn’t make sense

File: source/getstarted.html.erb (hunk 1 above)
Comment (jonheslop) [reply source/getstarted.html.erb]: ok will fix

File: source/getstarted.html.erb (hunk 2 above)
Comment (jonheslop) [reply source/getstarted.html.erb]: Will ask Tracy

Comment (alexbishop1) [reply source/getstarted.html.erb]: It makes perfect sense if the commas are removed.

Comment (jonheslop) [reply source/getstarted.html.erb]: changed it to 

> Use GOV.UK Pay's Payment Service Provider, Stripe.

**Pull request merged**